command/debug/debug.go

```diff
@@ -270,7 +270,8 @@ func (c *cmd) prepare() (version string, err error) {
 	// If none are specified we will collect information from
 	// all by default
 	if len(c.capture) == 0 {
-		c.capture = defaultTargets
+		c.capture = make([]string, len(defaultTargets))
```

Comment:
very fun slice debugging exercise where the code block below was mutating `defaultTargets` which affected tests running in parallel